coursedata/level-defaults/de.yaml

```diff
@@ -115,7 +115,7 @@
 4:
     start_code: "print 'Hallo Welt'"
     intro_text: |
-        In Level 4 `print` und `ask` ändern.
+        In Level 4 haben sich `print` und `ask` geändert.
 
         Du musst Text, den du genau so mit `print` ausgeben möchtest, in Anführungszeichen setzen.
 
```

Comment:
Want to fix the next line too?

Reply:
> Want to fix the next line too?

"Du musst Text, den du genau so mit `print` ausgeben möchtest, in Anführungszeichen setzen." seems already correct to me. What should be fixed there, and why would it belong in this PR rather than a separate one?

Reply:
122 Du kannst du

Reply:
Sorry, I assumed "next" meant the one after the already changed one, rather than the one after the location of your review comment (which the overview UI doesn't even show). I've not also fixed the grammar there and further below in the Level 4 intro, and also added changes to bring the Level 4 intro closer in meaning to its current English version.